From 864d70759357749a508ba0b5eb3f3fa0eea25170 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Thu, 4 Apr 2019 18:29:36 -0600
Subject: [PATCH] install modules into lib64

by default luarocks install rocks into /usr/lib/luarocks
but we go lib64, so make luarocks rock this way too.

Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>
---
 src/luarocks/core/cfg.lua | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/luarocks/core/cfg.lua b/src/luarocks/core/cfg.lua
index 3237786..5b57e78 100644
--- a/src/luarocks/core/cfg.lua
+++ b/src/luarocks/core/cfg.lua
@@ -187,8 +187,8 @@ local function make_defaults(lua_version, target_cpu, platforms, home)
       check_certificates = false,
 
       lua_modules_path = "/share/lua/"..lua_version,
-      lib_modules_path = "/lib/lua/"..lua_version,
-      rocks_subdir = "/lib/luarocks/rocks-"..lua_version,
+      lib_modules_path = "/lib64/lua/"..lua_version,
+      rocks_subdir = "/lib64/luarocks/rocks-"..lua_version,
 
       arch = "unknown",
       lib_extension = "unknown",
-- 
2.21.0

